README.md

```diff
@@ -1,14 +1,14 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S2-CS2103T-T17-3/tp/actions)
 
 ![Ui](docs/images/Ui.png)
```

Comment:
Do you think it is good to change this to our current mockup of the application?

Reply:
Yes, that would be ideal. Thank you for the suggestion!